src/current/v23.1/grant.md

```diff
@@ -138,32 +139,24 @@ SHOW GRANTS ON TABLE movr.public.*;
   database_name | schema_name |         table_name         | grantee | privilege_type | is_grantable
 ----------------+-------------+----------------------------+---------+----------------+---------------
   movr          | public      | promo_codes                | admin   | ALL            |      t
-  movr          | public      | promo_codes                | demo    | ALL            |      t
   movr          | public      | promo_codes                | max     | ALL            |      f
   movr          | public      | promo_codes                | root    | ALL            |      t
   movr          | public      | rides                      | admin   | ALL            |      t
-  movr          | public      | rides                      | demo    | ALL            |      t
   movr          | public      | rides                      | max     | ALL            |      f
-  movr          | public      | rides                      | max     | UPDATE         |      t
   movr          | public      | rides                      | root    | ALL            |      t
   movr          | public      | user_promo_codes           | admin   | ALL            |      t
-  movr          | public      | user_promo_codes           | demo    | ALL            |      t
   movr          | public      | user_promo_codes           | max     | ALL            |      f
   movr          | public      | user_promo_codes           | root    | ALL            |      t
   movr          | public      | users                      | admin   | ALL            |      t
-  movr          | public      | users                      | demo    | ALL            |      t
   movr          | public      | users                      | max     | ALL            |      f
   movr          | public      | users                      | root    | ALL            |      t
   movr          | public      | vehicle_location_histories | admin   | ALL            |      t
-  movr          | public      | vehicle_location_histories | demo    | ALL            |      t
   movr          | public      | vehicle_location_histories | max     | ALL            |      f
   movr          | public      | vehicle_location_histories | root    | ALL            |      t
   movr          | public      | vehicles                   | admin   | ALL            |      t
-  movr          | public      | vehicles                   | demo    | ALL            |      t
   movr          | public      | vehicles                   | max     | ALL            |      f
-  movr          | public      | vehicles                   | public  | SELECT         |      f
   movr          | public      | vehicles                   | root    | ALL            |      t
-(26 rows)
+(18 rows)
 ~~~
 
 To ensure that anytime a new table is created, all the privileges on that table are granted to a user, use [`ALTER DEFAULT PRIVILEGES`]({% link {{ page.version.version }}/alter-default-privileges.md %}):
```

Comment:
I cannot let this one pass

```suggestion
To ensure that any time a new table is created, all the privileges on that table are granted to a user, use [`ALTER DEFAULT PRIVILEGES`]({% link {{ page.version.version }}/alter-default-privileges.md %}):
```

Reply:
the language is evolving! the words are growing together! it's like 'email' (https://english.stackexchange.com/questions/1925/email-or-e-mail)

jkjk fixed it (for now ....)